joplin/js/admin.js

```diff
@@ -0,0 +1,10 @@
+import "../css/admin.scss";
```

Comment:
same stuff we saw with admin.css (except I moved a ton of the editor/preview logic out to their respective js files)